ONSdigital/rm-reporting
# update dependencies

# What and why?
Updating dependencies to resolve dependabot alerts

# How to test?
Do `pipenv install`, then `pipenv graph`, and check the following dependencies have upgraded:

- Flask-Cors (>=4.0.2)
- setuptools (>=70.0.0)
- gunicorn (>=22.0.0)
- urllib3 (>=2.2.2)
- requests (>=2.32.0)
- jinja2 (>=3.1.4)
- aiohttp (>=3.10.2)
- idna (>=3.7)
- certifii (>=2024.07.04)
- werkzeug (>=3.0.3)

Deploy, run tests, and make sure reporting service still works.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1304)
